### PR TITLE
deps.sh: Don't download dart-sdk if it exists

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Downloads and installs dart-sdk only if it doesn't exist already.

Fixes https://github.com/coala-analyzer/coala-bears/issues/157

### CODE REVIEW

File: .ci/deps.sh [hunk 1]
```diff
@@ -55,8 +55,10 @@ done
 python setup.py --help
 
 # Dart Lint commands
-wget -nc -O ~/dart-sdk.zip https://storage.googleapis.com/dart-archive/channels/stable/release/1.14.2/sdk/dartsdk-linux-x64-release.zip
-unzip -n ~/dart-sdk.zip -d ~/
+if [ ! -d "~/dart-sdk/" ] && [ ! -d "~/dart-sdk/bin" ]; then
```
Comment (AbdealiJK) [inline .ci/deps.sh]: Would it not be better to do `if ! dartlint -v ; then` as it will also check if the dartlint is installed correctly ?

Comment (Adrianzatreanu) [inline .ci/deps.sh]: https://github.com/coala-analyzer/coala-bears/issues/157#issuecomment-192712860

Comment (Adrianzatreanu) [inline .ci/deps.sh]: ah well i made it work now nvm

Comment (Adrianzatreanu) [inline .ci/deps.sh]: however, i think it would have a conflict with that? echo 'export PATH=$PATH:~/dart-sdk/bin' >> ~/.circlerc

Comment (AbdealiJK) [inline .ci/deps.sh]: I don't follow. This command runs after the export right ?

Comment (Adrianzatreanu) [inline .ci/deps.sh]: I dont know the order. You sure export runs first and then the download?

Comment (AbdealiJK) [inline .ci/deps.sh]: yep (Notice that the export is in the `pre` section in the config file

**Comment by Adrianzatreanu** [comment]
ok, do you like this now :) ?

**Comment by AbdealiJK** [comment]
ack 50cca07

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by jayvdb** [comment]
This didnt work at all.

**Pull request merged**